### DISCUSSION TITLE
Ability to 'Crop' a source target with knobs/faders

### DISCUSSION THREAD

**Discussion by gitagogaming** [lebaston100/MIDItoOBS#55] [Ideas]
I have a camera scene which I have to move around from time to time, depending on the persons camera feed I receive, this causes me to have to 'crop' and resize.. 

While I can resize/move everything with this program currently, I am unable to 'crop' it in any way.  
It would be great to have access to the Alt + Click/Drag to crop an item via midi

**Comment by lebaston100** [comment]
I will look into what can be done.

**Comment by lebaston100** [comment]
So it should be possible to set the crop for all sides. But this will move the scene item. So if the scene item is cropped by 10px from the left, the entire scene item will move 10px to the left.

**Comment by lebaston100** [comment]
Done https://github.com/lebaston100/MIDItoOBS/commit/b6a47138cb9e51ae3b6243e628f0df43037d9fbc
